applications/callflow/src/cf_exe.erl

```diff
@@ -106,6 +106,13 @@ continue(Key, Call) ->
     Srv = whapps_call:kvs_fetch('consumer_pid', Call),
     continue(Key, Srv).
 
+-spec continue_with_flow(wh_json:object(), whapps_call:call() | pid()) -> 'ok'.
```

Comment:
This function is not exported and not used internally. I assume you meant to export it?